doc/commands.rst

```diff
@@ -0,0 +1,78 @@
+| stopShutter     |           |                 |          |                                                 |
++-----------------+-----------+-----------------+----------+-------------------------------------------------+
+| config          | systemID  | | string        | unitless | | See the                                       |
+|                 |           |                 |          |   `Command and Configuration Protocols`_        |
+|                 | settings  | | [key, value]  | unitless |   document.                                     |
++-----------------+-----------+-----------------+----------+-------------------------------------------------+
+| park            |           |                 |          |                                                 |
++-----------------+-----------+-----------------+----------+-------------------------------------------------+
+| setTemperature  | temp      | | double        | Celsius  |                                                 |
++-----------------+-----------+-----------------+----------+-------------------------------------------------+
+| fans            | action    | | string        | unitless | ON or OFF                                       |
++-----------------+-----------+-----------------+----------+-------------------------------------------------+
+| inflate         | action    | | string        | unitless | ON or OFF                                       |
++-----------------+-----------+-----------------+----------+-------------------------------------------------+
+| statusAMCS      |           |                 |          |                                                 |
++-----------------+-----------+-----------------+----------+-------------------------------------------------+
+| statusApSCS     |           |                 |          |                                                 |
++-----------------+-----------+-----------------+----------+-------------------------------------------------+
+| statusLCS       |           |                 |          |                                                 |
++-----------------+-----------+-----------------+----------+-------------------------------------------------+
+| statusLWSCS     |           |                 |          |                                                 |
++-----------------+-----------+-----------------+----------+-------------------------------------------------+
+| statusMonCS     |           |                 |          |                                                 |
++-----------------+-----------+-----------------+----------+-------------------------------------------------+
+| statusThCS      |           |                 |          |                                                 |
++-----------------+-----------+-----------------+----------+-------------------------------------------------+
```

Comment:
If these tables are prone to change, maybe its better to make a CSV file and use the [csv-table directive](https://docutils.sourceforge.io/docs/ref/rst/directives.html#csv-table).

Reply:
In the end I chose the [list-table directive](https://docutils.sourceforge.io/docs/ref/rst/directives.html#list-table) since I needed to be able to span certain cells over two rows and I couldn't figure out how to do that with CSV.